.github/workflows/generator_generic_slsa3.yml

```diff
@@ -154,9 +163,36 @@ jobs:
           path: "${{ inputs.attestation-name }}"
           sha256: "${{ needs.generator.outputs.attestation-sha256 }}"
 
-      - name: Release
+      - name: Upload provenance
         uses: softprops/action-gh-release@1e07f4398721186383de40550babbdf2b84acfc5 # tag=v0.1.14
         id: release
         with:
           files: |
             ${{ inputs.attestation-name }}
```

Comment:
So when `upload-assets=true` and `upload-url` is set we upload twice? once as we do now (if this is a `workflow_dispatch` I guess this is a noop?), and once at the upload url specified?

Reply:
you''re correct. I've tried to not add much logic here, since the case you describe will hardly ever happen... I argue :)

But I can update if you think it's important.

Just let me know.

Reply:
Based on your other comments it seemed like `upload-url` would only have an effect if `upload-assets=true` so I'm a bit confused.

Just to be clear, we expect them to set one or the other, but not both?

Reply:
so we expect `upload-assets=true` to always be set, optionally with a url

Reply:
no longer relevant as per https://github.com/slsa-framework/slsa-github-generator/pull/713#discussion_r960293407